test/integration/universe/wizards-v1/universe-lifecycle/basic-paths/scenario-2.ts

```diff
@@ -0,0 +1,257 @@
+import { ERC721_WARPER_PRESET_IDS } from '../../../../../../src/protocol/warper/v1-controller/ERC721/constants';
+import { GENERAL_WIZARDS_HELPER_V1_ERRORS } from '../../../../../../src/protocol/wizards/helpers/general-v1/errors';
+import { EMPTY_BYTES4_DATA_HEX, EMPTY_BYTES_DATA_HEX } from '../../../../../../src/utils/bytes-and-hashing';
+import { MAX_TIME_PROTOCOL_UINT32 } from '../../../../../../src/utils/time';
+import {
+  ERC20Mock,
+  ERC721Mock,
+  GeneralWizardsHelperV1__factory,
+  IMetahub,
+  ITaxTermsRegistry,
+  IUniverseRegistry,
+  IUniverseRegistry__factory,
+  IUniverseWizardV1,
+  IWarperManager,
+  IWarperPresetFactory,
+  IWarperPresetFactory__factory,
+  Warpers__factory,
+} from '../../../../../../typechain';
+import { WarperDeployFromPresetBuilder } from '../../../../../builders/warper/warper-deploy-from-preset-builder';
+import { UniverseWithErc721WarperBuilderV1 } from '../../../../../builders/wizards-v1/universe/universe-erc721-warper-builder-v1';
+import { UniverseConstructorV1 } from '../../../../../constructors/universe/universe-constructor-v1';
+import { Erc721ConfigurableWarperConstructor } from '../../../../../constructors/warper/ERC721/erc721-configurable-warper-constructor';
+import { InvalidWarperConstructor } from '../../../../../constructors/warper/invalid-warper-constructor';
+import { EMPTY_ARRAY } from '../../../../../utils/common-test-stuff/dummy-data/general';
+import { COMMON_TEST_UNIVERSE_NAME_1 } from '../../../../../utils/common-test-stuff/dummy-data/universe';
+import { TEST_CASE, TEST_NOT_OK, TEST_OK } from '../../../../../utils/common-test-stuff/general-utils';
+import { TaxTermsRegistryValidator } from '../../../../../validators/tax/tax-terms-registry-validator';
+import { Erc721ConfigurablePresetValidator } from '../../../../../validators/warper/v1-controller/erc721-configurable-preset-validator';
+import { WarperManagerValidator } from '../../../../../validators/warper/warper-manager-validator';
+import { WarperPresetFactoryValidator } from '../../../../../validators/warper/warper-preset-factory-validator';
+import { UniverseValidatorV1 } from '../../../../../validators/wizards-v1/universe/universe-validator-v1';
+
+export const SCENARIO_2 = 'scenario-2';
+
+export const scenario2 = (): void => {
+  /**** Core Contracts ****/
+  let universeRegistry: IUniverseRegistry;
+  let universeWizardV1: IUniverseWizardV1;
+  let warperPresetFactory: IWarperPresetFactory;
+  let warperManager: IWarperManager;
+  let taxTermsRegistry: ITaxTermsRegistry;
+  let metahub: IMetahub;
+  /**** Mocks & Samples ****/
+  let baseToken: ERC20Mock;
+  let originalCollection: ERC721Mock;
+  /**** Signers ****/
+  let universeOwner: SignerWithAddress;
+  let stranger: SignerWithAddress;
+  /**** Data Structs ****/
+  let universeConstructor: UniverseConstructorV1;
+  let warperConstructor: Erc721ConfigurableWarperConstructor;
+  const taxTerms = makeTaxTermsFixedRateFromRawPercent('5');
```

Comment:
taxTerms -> universeWarperTaxTerms, since it reminds about specifics of the Tax Terms that we apply them to the given Universe -> registered Warper level.